Checks/PWR005/README.md

```diff
@@ -71,6 +73,58 @@ void example() {
 }
 ```
 
+#### Fortran
+
+In the following code, a variable `t` is used in each iteration of the loop to
+hold a value that is then assigned to the array `result`. Since no data scoping
+is declared for those variables, the default will be used. This makes the
+variable `t` shared, which is incorrect since it introduces a race condition:
+
+```f90
+subroutine example()
+  integer :: i, t
+  integer :: result(10)
+
+  !$omp parallel do
+  do i = 1, 10
+    t = i + 1;
+    result(i) = t;
+  end do
+end subroutine example
+```
+
+The following code disables the default scoping, which will make the compiler
+raise an error due to unspecified scopes:
+
+```f90
+subroutine example()
+  integer :: i, t
+  integer :: result(10)
+
+  !$omp parallel do default(none)
+  do i = 1, 10
+    t = i + 1;
+    result(i) = t;
+  end do
+end subroutine example
+```
+
+To fix the code, the scope of each variable must be specified. The variable `t`
+must be made private to prevent the race condition:
+
+```f90
+subroutine example()
+  integer :: i, t
+  integer :: result(10)
+
+  !$omp parallel do default(none) shared(result) private(t)
```

Comment:
Although not mandatory, I would recommend explicitly specifying the scope of i as well.

Reply:
I agree with @inaki-amatria on this scenario. See the thread above :)

[Done](https://github.com/codee-com/open-catalog/compare/70d218ee94a154404de7f2047bac58ee0689635a..53da9a7b92e35919f1543c12f7650952d5c94324)!